.buildkite/pipeline.yml

```diff
@@ -149,6 +159,7 @@ steps:
       FC_TEST_JAILER_BIN: "${FC_TEST_DATA_PATH}/jailer-main"
       DOCKER_IMAGE_TAG: "$BUILDKITE_BUILD_NUMBER"
     commands:
+      - "mkdir -p ${FC_TEST_DATA_PATH}"
```

Comment:
This already happens in the `prepare environment` step. Is it getting deleted somewhere along the way?